Source/QED/BreitWheelerEngineWrapper.hpp

```diff
@@ -0,0 +1,49 @@
+#ifndef WARPX_breit_wheeler_engine_wrapper_h_
+#define WARPX_breit_wheeler_engine_wrapper_h_
+
+#include "QedWrapperCommons.h"
+
+//BW ENGINE from PICSAR
+#include "breit_wheeler_engine.hpp"
+
+using WarpXBreitWheelerWrapper =
+    picsar::multi_physics::breit_wheeler_engine<amrex::Real, DummyStruct>;
+
+// Functors ==================================
+
+// These functors provide the core elementary functions of the library
+// Can be included in GPU kernels
+
+/* \brief Functor to initialize the optical depth of photons for the
+*   Breit-Wheeler process */
+class BreitWheelerGetOpticalDepth
+{
+public:
+    BreitWheelerGetOpticalDepth ()
+    {};
+
+    AMREX_GPU_DEVICE 
+    AMREX_FORCE_INLINE
+    amrex::Real operator() () const
+    {
+        return WarpXBreitWheelerWrapper::
+            internal_get_optical_depth(amrex::Random());
+    }
+};
+//____________________________________________
+
+// Factory class =============================
+
+/* \brief Wrapper for the Breit Wheeler engine of the PICSAR library */
```

Comment:
Ah one last comment. I am not very familiar with Doxygen, so correct me if I'm wrong, but is
```/* \brief Wrapper for the Breit Wheeler engine of the PICSAR library */```
a proper Doxygen formatting? Shoudn't it be e.g.,
```/** \brief Wrapper for the Breit Wheeler engine of the PICSAR library */```
?

Reply:
Have you had a change to check that?

Reply:
It _should_ work now, but I am not familiar with Doxygen too...